### DISCUSSION TITLE
2024 Cohort 2 - Group 20 Discussion

### DISCUSSION THREAD

**Discussion by sharon-digipres** [Digital-Preservation-Coalition/PythonStudyGroups#46] [General]
This is the discussion thread for those in Group 20 of the July-December 2024 Cohort

**Comment by GeoffBurnsy** [comment]
hello

**Comment by RubenVanDriesscheVO** [comment]
Discussed 17/07:
- Sofie will post a few resources and guides for the starters
- Our longer term goal will be able to write a script that is able to create checksums, write them to another file and validate them at a later date. 
- Short term for next sessions will be to install python, and read up on some resources to get a basic understanding of phyton.

**Comment by SofieRuysseveldtVO** [comment]
Hi all,

Below you find some useful external reading and learning materials:

- Al Sweigart, Automate the boring stuff with Python. Practical programming for total beginners (2020): https://automatetheboringstuff.com/
- Michael Dawson, Python programming for the absolute beginner (2010):
https://nibmehub.com/opac-service/pdf/read/Python%20Programming%20for%20the%20Absolute%20Beginner-%203rd%20Edition.pdf
- Python Tutorial with exercises, W3Schools: https://www.w3schools.com/python/default.asp
- Official Python Beginners Guide: https://wiki.python.org/moin/BeginnersGuide
- Bits and Bots Study Group: https://github.com/Lotte-W/Bits-and-Bots-study-group

All the best,

Sofie

**Comment by GeoffBurnsy** [comment]
Hi all

No one else from this group at the meeting today.  Do people think they'll be able to make the next one?
